### PR TITLE
Use a subscription instead of polling/refetching comments and sessions

### PR DESCRIPTION
**Pull request by hbenl**
To test this you need to run a local GraphQL API using Replayio/backend#5990 and point the app to that API by setting the URLs in `.env`, e.g.
```
NEXT_PUBLIC_API_URL=http://localhost:8081/v1/graphql
NEXT_PUBLIC_API_SUBSCRIPTION_URL=ws://localhost:8081/v1/graphql
```

### CODE REVIEW

File: src/ui/actions/comments.ts [hunk 1]
```diff
@@ -48,7 +48,7 @@ export function createComment(
 
     const response = await mutate({
       mutation: ADD_COMMENT_MUTATION,
-      refetchQueries: ["GetComments"],
+      // refetchQueries: ["GetComments"],
```
Comment (jasonLaster) [inline src/ui/actions/comments.ts:51]: Love this line

File: src/ui/hooks/recordings.ts [hunk 2]
```diff
@@ -158,20 +158,38 @@ export async function getRecording(recordingId: RecordingId) {
   return convertRecording(result.data?.recording);
 }
 
+let subscribed = false;
```
Comment (jasonLaster) [inline src/ui/hooks/recordings.ts:161]: could this be a `useState`

**Comment by markerikson** [comment]
Note that this will need to be rebased against `main` and updated to use Yarn instead

**Review by jaril — APPROVED** [review]
Seems to work well — LGTM!

**Pull request merged**